guides/source/active_record_querying.md

```diff
@@ -539,7 +539,7 @@ Client.order("orders_count ASC").order("created_at DESC")
 Selecting Specific Fields
 -------------------------
 
-By default, `Model.find` selects all the fields from the result set using `select *`.
+By default, finder queries will select all the fields from the result set using `select *`.
```

Comment:
My thoughts were here that, since the finder methods are defined near the top here: https://github.com/rails/rails/blame/master/guides/source/active_record_querying.md#L58, I went with that. 

Having said that, I'm happy to leave this as it was, or do you think I could perhaps better explain here that this is an example that is illustrative of the way all the finder methods work, and not just `Model.find`?